docs/pages/components/button.md

```diff
@@ -8,116 +8,67 @@ permalink: components/button.html
-<button class="fd-button fd-button--positive sap-icon--accept"></button>
-<button class="fd-button fd-button--negative sap-icon--decline"></button>
-{% endcapture %}
-{% include display-component.html component=button %}
+### Toggle Button
 
+Use toggle button in a toolbar to activate/deactivate item or element. Another user case is to switch between different states.
 
-## Standard button state
-The buttons can also be set to a state:
+### Segmented Button (previously known as Button Group)
 
-* **Normal**: The default state of the button. It can be clicked/tapped to perform the corresponding action.
-* **Selected**: Used to signal this button is selected among other buttons. This state can be rendered using `is-selected` class or `aria-selected="true"` attribute for accessibility.
-* **Disabled**: It cannot be clicked/tapped. This state can be rendered using `is-disabled` class or `aria-disabled="true"` attribute for accessibility.
-
-{% capture button-standard-state %}
-<button class="fd-button fd-button--emphasized">Normal State</button>
-<button class="fd-button fd-button--emphasized" aria-selected="true">Selected State</button>
-<button class="fd-button fd-button--emphasized" aria-disabled="true">Disabled State</button>
-<br><br>
-<button class="fd-button">Normal State</button>
-<button class="fd-button" aria-selected="true">Selected State</button>
-<button class="fd-button" aria-disabled="true">Disabled State</button>
-<br><br>
-<button class="fd-button fd-button--light">Normal State</button>
-<button class="fd-button fd-button--light" aria-selected="true">Selected State</button>
-<button class="fd-button fd-button--light" aria-disabled="true">Disabled State</button>
-<br><br>
-<button class="fd-button fd-button--standard">Normal State</button>
-<button class="fd-button fd-button--standard" aria-selected="true">Selected State</button>
-<button class="fd-button fd-button--standard" aria-disabled="true">Disabled State</button>
-<br><br>
-<button class="fd-button fd-button--positive">Normal State</button>
-<button class="fd-button fd-button--positive" aria-selected="true">Selected State</button>
-<button class="fd-button fd-button--positive" aria-disabled="true">Disabled State</button>
-<br><br>
-<button class="fd-button fd-button--negative">Normal State</button>
-<button class="fd-button fd-button--negative" aria-selected="true">Selected State</button>
-<button class="fd-button fd-button--negative" aria-disabled="true">Disabled State</button>
-{% endcapture %}
-
-{% include display-component.html component=button-standard-state %}
-
-
-# Segmented Button (previously Button Group)
-Group a series of buttons together on a single line with the segmented button.
+Group a series of buttons together on a single line with the segmented button. Only one of the options can be active, the others remain or become inactive. Pressing an option activates it. This type of buttons is comparable to a radio button group.
```

Comment:
Group a series of buttons together on a single line with the segmented button. Only one of the options can be active at a time, the others remain or become inactive. The option can be activated by clicking on it. This type of button is comparable to a radio button group.